action.json

```diff
@@ -11,7 +11,7 @@
 		"automation" :
 		{
 			"name": "automation",
-			"url": "<YOUR-CLOUD-FUNCTION-URL>"
+			"url": "https://us-central1-ponderosa-openhab.cloudfunctions.net/openhabGoogleAssistant"
```

Comment:
Please remove your specific stuffs and keep it generic.Thanks

Reply:
Wow, didn't even mean to commit this. Commits gone wild.